### PR TITLE
add solution

### PR DESCRIPTION
**Pull request by romafanasiev**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,9 +1,104 @@
...
+        born,
+        died,
+        fatherName,
+        motherName,
+        slug,
+        mother,
+        father,
+      };
+
+      return updatedPerson;
```
Comment (marina-tilniak) [inline src/App.tsx:49]:
```suggestion
      return {
        ...person,
        mother,
        father,
      };
```

File: src/App.tsx [hunk 2]
```diff
@@ -1,9 +1,104 @@
...
-  <div className="App">
-    <h1>People table</h1>
-  </div>
-);
+const App: React.FC = () => {
+  const [people, setPeoples] = useState<People[]>([]);
+  const [updatedPeople, setUpdatedPeople] = useState<UpdatedPersons[]>([]);
+
+  const addParents = (persons: People[]) => {
+    const updatedPersons = persons.map((person) => {
```
Comment (marina-tilniak) [inline src/App.tsx:18]:
```suggestion
    return persons.map((person) => {
```

File: src/App.tsx [hunk 3]
```diff
@@ -1,9 +1,104 @@
...
+        motherName,
+        slug,
+        mother,
+        father,
+      };
+
+      return updatedPerson;
+    });
+
+    return updatedPersons;
```
Comment (marina-tilniak) [inline src/App.tsx:52]: delete it

File: src/App.tsx [hunk 4]
```diff
@@ -1,9 +1,104 @@
...
+        <nav>
+          <NavLink to="/">Home</NavLink>
+          <br />
+          <NavLink to="/people">People</NavLink>
+        </nav>
+      </header>
+
+      <div className="App">
+        <Routes>
+          {/* <Route path="/people" element={<p>People page</p>} /> */}
```
Comment (marina-tilniak) [inline src/App.tsx:86]: delete commented code

File: src/components/PersonRow/PersonRow.tsx [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+export const PeopleRow: React.FC<Props> = ({ person }) => {
+  const {
+    name, sex, born, died, fatherName, motherName, slug,
+  } = person;
+
+  return (
+    <tr
+      className="Person"
+      key={slug}
```
Comment (marina-tilniak) [inline src/components/PersonRow/PersonRow.tsx:15]: do you need key here?

**Review by marina-tilniak — CHANGES_REQUESTED** [review]

**Review by aliceblunt8 — APPROVED** [review]

**Pull request closed without merging**